src/probnum/quad/solvers/belief_updates/_belief_update.py

```diff
@@ -51,20 +57,68 @@ def __call__(
         updated_state :
             Updated version of ``bq_state`` that contains all updated quantities.
         """
+        raise NotImplementedError
+
+    def _gram_cholesky_decomposition(self, gram: np.ndarray) -> np.ndarray:
+        """Find the Cholesky decomposition of a positive-definite Gram matrix.
+
+        Parameters
+        ----------
+        gram :
+            symmetric pos. def. kernel Gram matrix :math:`K`, shape (nevals, nevals)
+
+        Returns
+        -------
+        chol_gram :
+            The Cholesky decomposition of the Gram matrix.
+        """
+        chol_gram = cho_factor(gram + self.jitter * np.eye(gram.shape[0]))
+        return chol_gram
+
+
+class BQStandardBeliefUpdate(BQBeliefUpdate):
+    """Updates integral belief and state using standard Bayesian quadrature based on
+    standard Gaussian process inference.
+
+    Parameters
+    ----------
+    jitter
+        Non-negative jitter to numerically stabilise kernel matrix inversion.
+    scale_estimator
+        Estimator to use to compute the scale parameter.
+    """
+
+    def __init__(self, jitter: FloatLike, scale_estimator: str) -> None:
```

Comment:
As per `bayesquad`, `scale_estimator` could be `None` as well (I suppose meaning no update?). 
```suggestion
    def __init__(self, jitter: FloatLike, scale_estimator: Optional[str]) -> None:
```